workers/unity/Packages/com.improbable.gdk.core/Worker/WorkerBase.cs

```diff
@@ -1,108 +1,158 @@
 using System;
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core.Components;
 using Improbable.Worker;
 using Unity.Entities;
 using UnityEngine;
+using Entity = Unity.Entities.Entity;
 
 namespace Improbable.Gdk.Core
 {
     public abstract class WorkerBase : IDisposable
     {
-        public World World { get; private set; }
-
-        public MutableView View { get; private set; }
-
         public Connection Connection { get; private set; }
 
-        public Vector3 Origin { get; private set; }
+        public Vector3 Origin { get; }
```

Comment:
I think this still probably wants a `setter` somehow - origin moving is definitely something that we will need to support

Reply:
Maybe out of scope for this PR - but something to keep in mind

Reply:
yes, will keep it in mind for the next PR. as said: the worker class is currently a mess and needs to be cleaned up